examples/js/loaders/DRACOLoader.js

```diff
@@ -198,7 +198,18 @@ THREE.DRACOLoader.prototype = Object.assign( Object.create( THREE.Loader.prototy
 
 		// Remove task from the task list.
 		geometryPending
-			.finally( () => {
+			.then( () => {
+
+				if ( worker && taskID ) {
+
+					this._releaseTask( worker, taskID );
+
+					// this.debug();
+
+				}
+
+			} )
+			.catch( () => {
```

Comment:
Note that you can condense this to avoid duplicating the cleanup logic:

```
geometryPending
  .catch( () =>  true )
  .then( () => {
    // ...
  } );
```

A `.then()` block after a `.catch()` will still be called in error cases, assuming the `.catch()` block handles the error without throwing a new error itself. https://stackoverflow.com/a/42028776/1314762

Reply:
^It would also be worth including a short comment explaining *why* we're doing something confusing like that, or future readers may wonder why we didn't just use `.finally()`. 😇

Reply:
I actually did not know that, thanks. TIL.

> It would also be worth including a short comment explaining why we're doing something confusing like that

Good point, will do.

Reply:
Tested locally and `then()` block is indeed called correctly after `catch()`. This should be good, unless we have a better suggestion for the comment.